### ISSUE TITLE
Request for 2021 Update

### ISSUE THREAD

**Issue by silvaDominic** [mpbarlow/vue-renderless-carousel#3]
I'm always keen on using renderless component plugins as I don't have to hack/override the source code. This was probably the only carousel plugin I could find that looked respectable and I would like to continue using it in the future. However, with it not being updated in 2 years I feel a little uneasy using it in a production project. Anyway you can assure us this isn't an abandoned repo and provide any compatibility updates?

**Comment by mpbarlow** [comment]
Hi. Thanks for your interest. To be perfectly honest, while I would like to one day do a 2.0 with Vue 3 support (and probably only Vue 3 support—it doesn’t look like interoperability is particularly easy) and some other improvements, I have no use for this library in my work anymore. As such, any future development would be done under the pretence of it being a fun project, so I can’t make any guarantees on when I would even start. 

Because of this I would probably advise you don’t use this for a new production project. That said, the code itself is pretty simple (two components and a single dependency, which itself probably has a browser-native alternative by now), so you if wanted to fork it or even just copy the components into your project, it could make a good starting point without introducing tons of complexity.

**Comment by silvaDominic** [comment]
Thanks for the reply @mpbarlow . Wasn't sure I'd be able to get to you.

Fortunately, I do not have any feature requests so I'm comfortable there. Converting our project to Vue3 is somewhat down the line and will be a whole task in itself. I'd likely implement our own version then. I was more concerned if there are any updates that need/can be made in regards to Vue2. I had set up a PoC carousel in our project before your response and it _appears_ to be working just fine and, as you said, there's only 1 dependency so less room for bugs. Just don't know if there's _anything_ that could be amiss since last update was so long ago.

**Comment by mpbarlow** [comment]
No worries. Well in that case, I can give it a once over some time this week? I don’t know of any major bugs, but I’m happy to update the dependency, see if I can get #2 fixed, and make sure everything is still working nicely. 

I’ll let you know when I’ve tagged a new version.

**Comment by silvaDominic** [comment]
@mpbarlow A true gentleman! That'd be so great and is much appreciated...And take it for what it's worth, but I do think you are just about the _only_ renderless carousel plugin out there.

**Comment by mpbarlow** [comment]
You're too kind 😄 Yeah, when I originally wrote this I couldn't find anything that fit my needs which is why I ended up writing my own.

I've just tagged 1.3.0. It updates the dependency to its latest version, and I also spotted an easy performance win—namely the resize handler will now only be called at most 60 times per second instead of potentially hundreds.

Let me know if you run into any issues with the new version—I've tested it against a test harness project, but unfortunately when I first wrote this I didn't really know much about testing Vue, so I never got any unit testing set up.
